int64 taskIDFrom
int64 robotIDFrom
int64 taskIDTo
int64 robotIDTo
int64 dependencyType